### ISSUE TITLE
FindingsReport outputs strange character with MaxLineLength

### ISSUE THREAD

**Issue by TWiStErRob** [detekt/detekt#4927]
## Observed Behavior
```
....kt - 20min debt
        MaxLineLength - [Line detected, which is longer than the defined maximum line length in the code (à)] at ....kt:138:4
```

## Expected Behavior
```
....kt - 20min debt
        MaxLineLength - [Line detected, which is longer than the defined maximum line length in the code style.] at ....kt:138:4
```
or
```
....kt - 20min debt
        MaxLineLength - [Line detected, which is longer than the defined maximum line length in the code style (120).] at ....kt:138:4
```

## Steps to Reproduce
Use `FindingsReport` or `FileBasedFindingsReport`;

it does NOT repro with `LiteFindingsReport`:
```
....kt:99:2: Line detected, which is longer than the defined maximum line length in the code style. [MaxLineLength]
```

Note sure if this part is relevant, but:
```groovy
ignoreFailures = true
buildUponDefaultConfig = false
allRules = true
```
```yml
config:
  warningsAsErrors: true
processors:
  active: false
console-reports:
  active: true
  exclude:
    - 'NotificationReport'
#    - 'FindingsReport'
    - 'FileBasedFindingsReport'
    - 'LiteFindingsReport'
```

## Context
Just configuring Detekt, and notice something was off.

Note: I tried to prove it was ASCII character 120, but changing to
```
style:
  MaxLineLength: 
    maxLineLength: 65
```
didn't yield `(A)` as I expected.

## Your Environment

* Version of detekt used: 1.20.0
* Version of Gradle used (if applicable): 7.4.2
* Gradle scan link (add `--scan` option when running the gradle task):
* Operating System and version: Windows 10, PowerShell & cmd
* Link to your project (if it's a public repository): N/A

**Comment by 3flex** [comment]
Thanks for the hint about which reporters are causing this - the two affected reports use this function which attempts to render in yellow (line 37):
https://github.com/detekt/detekt/blob/e8e1f35dfc303858651571624baa6df22ae0f8dc/detekt-core/src/main/kotlin/io/gitlab/arturbosch/detekt/core/reporting/Reporting.kt#L26-L45

Which terminal are you using? Does it have color support?

**Comment by TWiStErRob** [comment]
> Operating System and version: Windows 10, PowerShell & cmd

and yes it does
![image](https://user-images.githubusercontent.com/2906988/174337922-bc24139e-b07e-4e37-98d3-49fa7cd373de.png)

**Comment by 3flex** [comment]
Thanks - colorization was a red herring, it's actually a text encoding problem. detekt assumes the terminal supports Unicode, but Windows uses windows-1252 output encoding by default, including in PowerShell. I've opened a PR to fix.
